tests/conftest.py

```diff
@@ -96,7 +108,9 @@ def app_config(app_config):
     https://github.com/inveniosoftware/invenio-records-permissions/issues/51
     """
     app_config["RECORDS_REST_ENDPOINTS"] = {}
-
+    app_config["INDEXER_DEFAULT_DOC_TYPE"] = "testrecord"
+    app_config["INDEXER_DEFAULT_INDEX"] = TestSearch.Meta.index
+    app_config["SEARCH_MAPPINGS"] = ['drafts', 'records']
```

Comment:
minor: not necessary right? (we don't do it in invenio-records-resources)

Reply:
I did it for completeness, this avoids an issue if some other app instantiated indexes (with same alias might still somehow class).